samples/peripheral/802154_phy_test/README.rst

```diff
@@ -0,0 +1,357 @@
+     - custom lgettemp
+   * - Transition radio to sleep mode
+     - custom lsleep
+     - 
+     - CMD will put radio into sleep mode
+     - custom lsleep
+   * - Transition radio to receive mode
+     - custom lreceive
+     - 
+     - CMD will put radio in receive mode
+     - custom lreceive
+   * - Reboot the device
+     - custom lreboot
+     - 
+     - Reboots the device
+     - custom lreboot
+
+
+   
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/802154_phy_test`
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+   On the |nRF5340DKnoref|, the IEEE 802.15.4 PHY Test Tool is a standalone network sample that does not require any counterpart application sample.
+   However, you must still program the application core to boot up the network core.
+   You can use any sample for this, for example :ref:`nrf5340_empty_app_core`.
+   The :ref:`nrf5340_empty_app_core` is built and programmed automatically by default.
+   If you want to program another sample for the application core, unset the :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE' option.
+
+.. _802154_phy_test_testing:
+
+Using the tool
+==============
+
+After programming the supported development kit of your choice with the sample, you can verify that it functions as expected and begin performing tests.
+
+.. _802154_phy_test_testing_check:
+
+Checking if the sample works as expected
+----------------------------------------
+
+1. Connect the development kit to the computer using a USB cable. Use the DK's programmer USB port, not the nRF USB port located near the SoC.
+   The kits are assigned a COM port (Windows) or ttyACM device (Linux), which should be now visible in the Device Manager or /dev directory.
+#. Connect to the DK using a terminal emulator (for example, PuTTY or minicom), with the following settings:
+
+    - Baudrate: 115200
+    - Parity bits: none
+    - Stop bits: 1
+
+#. If the sample is configured to support BOTH modes (the default setting), switch the DK into CMD mode by sending the following command:
+
+    - ``custom changemode 1``
+
+#. On the bottom side of your DK, locate a table describing GPIO pin assignment to LEDs. Read the number of GPIO pin assigned to LED 1, 2, 3 or 4. For example, on nRF52840DK, the LEDs are controlled by pins from P0.13 through P0.16.
+#. The LEDs on nRF5340DK and nRF52840DK are in ``sink`` configuration. In order to turn them on, you need to set the respective pin's state to low in order to let the current flow through the LED. Send the following command to the device:
+    
+    - ``custom lsetgpio <pin> 0`` where ``<pin>`` is the number of the pin assigned for selected LED.
+    - ``custom lsetgpio 28 0`` would be the proper command to light up LED 1 on nRF5340DK. 
+
+#. If the selected LED lights up, the sample works as expected and is redy for use. 
+
+.. _802154_phy_test_testing_board:
+
+Testing with another development kit
```

Comment:
DUT mode is created to perform tests for devices without serial interface. I would rename this chapter to `Performing radio tests without serial interface` or something like this. 

You can perform tests using two development kits in CMD mode with commands like `lstart`, 'ltx'.

Reply:
Thank you, I will attempt to fix the problems right away.